robot.py

```diff
@@ -69,8 +69,8 @@ def createObjects(self):
         """Create motors and stuff here."""
 
         # a + + b - + c - - d + -
-        x_dist = 0.2625
-        y_dist = 0.2165
+        x_dist = 0.255
+        y_dist = 0.26
```

Comment:
So I just double-checked the CAD. x should be 525 / 2 = 262.5 mm, y should be 533 / 2 = 266.5 mm.